src/sass/app/trade.scss

```diff
@@ -668,6 +669,28 @@
             width: 175px;
         }
     }
+    // fix trading chart on small screens
+    .chart-dialog {
+        min-width: 300px !important;
+
+        .chartOptions {
+            margin: 0 !important;
+            position: relative;
+            top: 20px;
+            z-index: 1;
+
+            .table.justified-center + .instrument_name {
+                position: relative;
+                top: 15px;
+            }
+        }
+        svg.highcharts-root {
+            width: 100vw !important;
+            margin: 0 -10px !important;
+            position: relative;
+            left: -10px;
+        }
+    }
```

Comment:
these are all elements inside the webtrader chart container, shouldn't they be added to webtrader repo instead of binary-static? @4p00rv please comment

Reply:
@negarn I agree. 
@ruskakimov can you please submit a separate PR to [webtrader-charts](https://github.com/binary-com/webtrader-charts) for the fix.